odoo/tests/common.py

```diff
@@ -76,6 +76,23 @@ def get_db_name():
     return db
 
 
+
+def make_test_registry():
+    registry = {}
+    def register(func):
+        # determine key, by odoo module name otherwise by function name
+        if func.__module__.startswith('odoo.addons.'):
+            key = func.__module__.split('.')[2]
+        else:
+            key = func.__name__
+
+        registry[key] = func
+        return func
+    register.registry = registry
+    return register
+post_testing = make_test_registry()
```

Comment:
The name `post_testing` is confusing, as we already have "post-install" tests.  Those are standalone test scripts.  I suggest to use `standalone` for the decorator's name (or `script`).  I am open to alternative names, if you have some.  Second, you cannot have more than one test per module.  Third, you cannot name individual tests if you have more than one per module.

I suggest the alternative API, where you can tag the test with any number of names.  For the sake of simplicity, the only implicit tag is the module name.
```python
@odoo.tests.common.standalone('l10n_all')  # tagged with 'account' and 'l10n_all'
def test_l10n_all(env):
    ...
```
The implementation can be simpler, too.  I don't understand the point of `make_test_registry` if you call it once.
```python
standalone_tests = defaultdict(list)

def standalone(*tags):
    def register(func):
        # register func by odoo module name
        if func.__module__.startswith('odoo.addons.'):
            module = func.__module__.split('.')[2]
            standalone_tests[module].append(func)
        for tag in tags:
            standalone_tests[tag].append(func)
        return func

    return register
```

Reply:
done, I also made it so all tests are recorded to the key 'all', makes it easier for the part that handles user input `./test_module_operations.py -d foo --standalone all` will just lookup all in the dict